tests/openvino/test_seq2seq.py

```diff
@@ -652,12 +654,21 @@ def test_compare_to_transformers(self, model_arch):
             transformers_inputs["past_key_values"] = DynamicCache()
 
         with torch.no_grad():
+            tokenizer = None
+            if model_arch in ["minicpmo"]:
+                # `generate` method for minicpmo requires tokenizer
+                tokenizer = AutoTokenizer.from_pretrained(
```

Comment:
let's rather add it to `additional_inputs`

Reply:
Done